deltachat-ffi/deltachat.h

```diff
@@ -5120,11 +5132,6 @@ void dc_event_unref(dc_event_t* event);
 /// Used to build the string returned by dc_get_contact_encrinfo().
 #define DC_STR_E2E_AVAILABLE              25
 
-/// "Transport-encryption."
-///
-/// Used to build the string returned by dc_get_contact_encrinfo().
-#define DC_STR_ENCR_TRANSP                27
```

Comment:
ahh, wait, better leave the define but replace the doxygen-part by `// DEPRECATED 2020-02-06` or so  - this kind of removing always causes troubles  when using current master eg. on node/desktop ...